### ISSUE TITLE
Version 20220621 started failing on S390X recently

### ISSUE THREAD

**Issue by SnarkBoojum** [JohnCremona/eclib#73]
The same version which [was working](https://buildd.debian.org/status/package.php?p=eclib&suite=bookworm) is [now failing](https://buildd.debian.org/status/fetch.php?pkg=eclib&arch=s390x&ver=20220621-2&stamp=1664757252&raw=0). I'm not sure exactly why, but since pari got updated in Debian recently, it's a possible explanation.

Since it's a timeout thing, perhaps something takes a little more time and that's enough to break... how can I help debug the issue?

**Comment by JohnCremona** [comment]
Looking at the log, the only test which starts but does not finish is tsatbnd (looking for "Testing abc..." with no "abc completed", taking into account that they can be interleaved).   It is true that the saturation code (which this tests) was changed quite a lot in 2021 (in fact as a result of saturation now being also implemented in libpari -- though that is not used here).

During those changes I added one more call to libpari; before this it only used libpari for integer factorization, while now it also uses the ellap() and ellinit() functions, and these are called during the saturation process.

Is it possible to see the partial output file tsatbnd.testout whcih would be created in the tests/ directory (but normally deleted at the end of the tests)?  It seems likely that the sticking point is in the new large test I added adfter making the saturation process much more efficient...
